content/docs/installation/best-practice.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Best Practice
+description: Learn how to deploy cert-manager to comply with popular security standards such as CIS, NSA, and BSI.
+---
+
+Learn how to deploy cert-manager to comply with popular security standards such as
+the [CIS Kubernetes Benchmark](https://www.cisecurity.org/benchmark/kubernetes/),
+the [NSA Kubernetes Hardening Guide](https://media.defense.gov/2022/Aug/29/2003066362/-1/-1/0/CTR_KUBERNETES_HARDENING_GUIDANCE_1.2_20220829.PDF), or
+the [BSI Kubernetes Security Recommendations](https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Grundschutz/International/bsi_it_gs_comp_2022.pdf?__blob=publicationFile&v=2#page=475).
+
+## Overview
+
+The default cert-manager resources in the Helm chart or YAML manifests ( Deployment, Pod, ServiceAccount etc) are designed for backwards compatibility rather than for best practice or maximum security.
```

Comment:
```suggestion
The default cert-manager resources in the Helm chart or YAML manifests (Deployment, Pod, ServiceAccount etc) are designed for backwards compatibility rather than for best practice or maximum security.
```